### ISSUE TITLE
Ambiguous 'tags'

### ISSUE THREAD

**Issue by irees** [transitland/transitland-datastore#619]
When using a 'tags' controller filter, ambiguous column during certain joins.

```[PG::AmbiguousColumn] ERROR:  column reference "tags" is ambiguous
LINE 1: ...= "current_feeds"."active_feed_version_id" WHERE (tags -> 'd...
                                                             ^
```

**Comment by drewda** [comment]
@irees duplicate of #612 ?

**Comment by irees** [comment]
Closed as duplicate of #612
